allennlp/data/dataset_readers/conll2003.py

```diff
@@ -95,12 +103,22 @@ def __init__(
         for label in feature_labels:
             if label not in self._VALID_LABELS:
                 raise ConfigurationError("unknown feature label type: {}".format(label))
-        if coding_scheme not in ("IOB1", "BIOUL"):
-            raise ConfigurationError("unknown coding_scheme: {}".format(coding_scheme))
+        if convert_to_coding_scheme not in (None, "BIOUL") or coding_scheme not in (
+            "IOB1",
+            "BIOUL",
+        ):
+            raise ConfigurationError("unknown coding_scheme: {}".format(convert_to_coding_scheme))
```

Comment:
I think we should separate these cases to provide a better error message.

```suggestion
        if convert_to_coding_scheme not in (None, "BIOUL"):
            raise ConfigurationError(f"invalid convert_to_coding_scheme: {convert_to_coding_scheme}")
        if coding_scheme not in ("IOB1", "BIOUL"):
            raise ConfigurationError(f"invalid coding_scheme: {coding_scheme}")
```